tests/networks.spec.ts

```diff
@@ -4,148 +4,378 @@ import {
+    // network names can only be 15 characters long
+    network = `${browserName.substring(0, 2)}-${network}`;
+    const page = await browser.newPage();
+    await page.goto("/ui/settings");
+
+    const parent = page.getByRole("row").filter({
+      hasText: "network.ovn.northbound_connection",
+    });
+    await parent.getByRole("button").click();
+
+    initialNorthboundNetworkValue = await parent
+      .getByRole("textbox")
+      .inputValue();
+
+    await parent.getByRole("textbox").fill("foo");
+
+    await page
+      .getByRole("button", {
+        name: /save/i,
+      })
+      .click();
+
+    await page.close();
+  });
+
+  test.afterAll(async ({ browser }) => {
+    const page = await browser.newPage();
+    await page.goto("/ui/settings");
+
+    const parent = page.getByRole("row").filter({
+      hasText: "network.ovn.northbound_connection",
+    });
+    await parent.getByRole("button").click();
+    await parent.getByRole("textbox").fill(initialNorthboundNetworkValue);
+    await page
+      .getByRole("button", {
+        name: /save/i,
+      })
+      .click();
+
+    await page.close();
+  });
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/ui/");
+    await page.getByRole("link", { name: "Networks", exact: true }).click();
+    await page.getByRole("button", { name: "Create network" }).click();
+    await page.getByRole("heading", { name: "Create a network" }).click();
+    await page.getByLabel("Type").selectOption("OVN");
+    await page.getByLabel("Name").click();
+    await page.getByLabel("Name").fill(network);
+    await page.getByLabel("Uplink").selectOption({ index: 2 });
+  });
+
+  test("configure main OVN network settings", async ({ page }) => {
+    const options = ["false", "true"];
+
+    await page
+      .getByPlaceholder("Enter description")
+      .fill(NETWORK_CONSTANTS.DESCRIPTION);
+
+    await page
+      .getByRole("row", {
+        name: "IPv4 address IPv4 address for",
+      })
+      .getByRole("button")
+      .click();
+    await page
+      .getByRole("row", {
+        name: "IPv6 address IPv6 address for",
+      })
+      .getByRole("button")
+      .click();
+
+    const radioButtons = await page.$$("input[type=radio]");
+    for (const button of radioButtons) {
+      await button.click({ force: true });
+    }
+
+    await page.getByLabel("IPv4 address").fill(NETWORK_CONSTANTS.IPV4_ADDRESS);
+    await page.getByLabel("IPv6 address").fill(NETWORK_CONSTANTS.IPV6_ADDRESS);
+
+    for (const option of options) {
+      await setOption(page, "IPv4 NAT", option);
+      await setOption(page, "IPv6 NAT", option);
+    }
+  });
+
+  test("set OVN bridge MTU and hardware address", async ({ page }) => {
+    await page
+      .getByRole("button", {
+        name: "Advanced",
+      })
+      .click();
+    await page
+      .getByLabel("Network form navigation")
+      .getByText("Bridge", { exact: true })
+      .click();
```

Comment:
```suggestion
    await visitNetworkConfiguration(page, "Bridge");
```
This is duplicated quite a few times, maybe worth adding a helper function `visitNetworkConfiguration` Maybe we can even reuse the existing `prepareNetworkTabEdit`?